test/WebSites/InlineConstraintsWebSite/Constraints/IsbnDigitScheme13Constraint.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class IsbnDigitScheme13Constraint : IRouteConstraint
```

Comment:
I believe ISBN13 also has optional dashes. At least the examples I found seemed to sometimes have dashes (but with 5 segments, instead of 4).